pyembree/rtcore_scene.pyx

```diff
@@ -19,12 +23,21 @@ cdef class EmbreeScene:
 
     def run(self, np.ndarray[np.float64_t, ndim=2] vec_origins,
                   np.ndarray[np.float64_t, ndim=2] vec_directions, 
-                  dists=None):
+                  dists=None,query='INTERSECT'):
         rtcCommit(self.scene_i)
         cdef int nv = vec_origins.shape[0]
         cdef int vo_i, vd_i, vd_step
         cdef np.ndarray[np.int32_t, ndim=1] intersect_ids
         cdef np.ndarray[np.float32_t, ndim=1] tfars
+        cdef rayQueryType query_type
+        
+        if query == 'INTERSECT':
+            query_type = intersect
+        elif query == 'OCCLUDED':
+            query_type = occluded
```

Comment:
For what it's worth, yt uses pyembree and still supports Python2.7, so it would be in our interests to keep it there for a little while longer.  There are other bits in pyembree that use strings, IIRC.